software/base/src/main/java/org/apache/brooklyn/entity/software/base/SoftwareProcessImpl.java

```diff
@@ -85,15 +89,23 @@
  * It exposes sensors for service state (Lifecycle) and status (String), and for host info, log file location.
  */
 public abstract class SoftwareProcessImpl extends AbstractEntity implements SoftwareProcess, DriverDependentEntity {
-    private static final Logger log = LoggerFactory.getLogger(SoftwareProcessImpl.class);
-    
+
+    private static final Logger LOG = LoggerFactory.getLogger(SoftwareProcess.class);
```

Comment:
Did you remove the `Impl` suffix on purpose? I think we should keep it - it points to the file where the logging occurs.